### PR TITLE
Support stackprof's ignore_gc option

### PR DESCRIPTION
**Pull request by swalkinshaw**
Allows the `ignore_gc` boolean option and passes it through to stackprof.

> By default, samples taken during garbage collection will show as garbage collection frames including both mark and sweep phases. For longer traces, these can leave gaps in a flamegraph that are hard to follow. They can be disabled by setting the ignore_gc option to true. Garbage collection time will still be present in the profile but not explicitly marked with its own frame.

### CODE REVIEW

**Review by jasonhl — APPROVED** [review]
One nitpick which you can choose to address or ignore.

File: lib/app_profiler/request_parameters.rb [hunk 1]
```diff
@@ -50,6 +51,11 @@ def mode
       query_param("profile") || profile_header_param("mode")
     end
 
+    def ignore_gc
+      value = query_param("ignore_gc") || profile_header_param("ignore_gc")
+      value == "true"
```
[the remark this answers is not in the document]
Comment (swalkinshaw) [reply lib/app_profiler/request_parameters.rb]: That's actually more consistent with how `autoredirect` works so I updated it 👍

**Review by gmcgibbon — APPROVED** [review]
